LMChatCore_iOS/LMChatCore_iOS/Source/Module/Poll/AddOption/LMChatPollAddOptionScreen.swift

```diff
@@ -0,0 +1,247 @@
+//
+//  LMChatPollAddOptionScreen.swift
+//  LikeMindsChatCore
+//
+//  Created by Pushpendra Singh on 26/07/24.
+//
+
+import LikeMindsChatUI
+import UIKit
+
+public protocol LMChatAddOptionProtocol: AnyObject {
+    func onAddOptionResponse(postID: String, success: Bool, errorMessage: String?)
+}
+
+open class LMChatPollAddOptionScreen: LMViewController {
+    // MARK: UI Elements
+    open private(set) lazy var dismissView: LMView = {
+        let view = LMView().translatesAutoresizingMaskIntoConstraints()
+        view.backgroundColor = Appearance.shared.colors.black.withAlphaComponent(0.5)
+        return view
+    }()
+    
+    open private(set) lazy var containerView: LMView = {
+        let view = LMView().translatesAutoresizingMaskIntoConstraints()
+        view.backgroundColor = Appearance.shared.colors.white
+        return view
+    }()
+    
+    open private(set) lazy var sepratorView: LMView = {
+        let view = LMView().translatesAutoresizingMaskIntoConstraints()
+        view.backgroundColor = Appearance.shared.colors.backgroundColor
+        return view
+    }()
+    
+    open private(set) lazy var titleLabel: LMLabel = {
+        let label = LMLabel().translatesAutoresizingMaskIntoConstraints()
+        label.font = Appearance.shared.fonts.headingFont1
+        label.text = "Add new poll option"
+        label.textColor = Appearance.shared.colors.gray51
+        return label
+    }()
+    
+    open private(set) lazy var subtitleLabel: LMLabel = {
+        let label = LMLabel().translatesAutoresizingMaskIntoConstraints()
+        label.font = Appearance.shared.fonts.subHeadingFont2
+        label.text = "Enter an option that you think is missing in this poll. This can not be undone."
```

Comment:
Can we move the hardcoded strings to constant file?

Reply:
This file no more use